resources/lib/infolabels.py

```diff
@@ -201,10 +201,18 @@ def InfoLabel_IsScreenSaverActive():
 def InfoLabel_IsMuted():
   return InfoLabel_GetBool("Player.Muted")
 
+def InfoLabel_IsVolumeChanging():
```

Comment:
You can check/monitor lcdbase's m_bVolumeChangeActive for this instead, saves the call to Kodi's internals.